app/src/main/java/com/example/peterchu/watplanner/Models/Schedule/CourseComponent.java

```diff
@@ -212,11 +218,43 @@ public String toString() {
         return String.format("%s %s - %s %s - %s %s~%s", subject, catalogNumber, type, section, day, startTime, endTime);
     }
 
+    private Calendar getTermStartDate() {
```

Comment:
hardcoded method until we lookup date info from the term id, which we can do later